app/assets/stylesheets/global.css.scss

```diff
@@ -0,0 +1,125 @@
+$dark_gray: #2B2E2E;
+$light_gray: #C0C1C1;
+$green: #33CD99;
+
+@import url(http://fonts.googleapis.com/css?family=Amaranth:700italic);
+
+@mixin bold_font {
```

Comment:
Name of mixin `bold_font` should be written in all lowercase letters with hyphens instead of underscores